src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/HttpResourceManagerRpc.java

```diff
@@ -301,4 +303,59 @@ public Policy replacePolicy(String projectId, Policy newPolicy) throws ResourceM
       throw translate(ex);
     }
   }
+
+  @Override
+  public Lien createLien(Lien lien) {
+    try {
+      return resourceManager.liens().create(lien).execute();
+    } catch (IOException ex) {
+      ResourceManagerException translated = translate(ex);
+      if (translated.getCode() == HTTP_FORBIDDEN) {
+        return null;
```

Comment:
Why is this case special? Why not an exception like every other case?

Reply:
done

Reply:
done